rust/tw_evm/src/transaction/user_operation_v0_7.rs

```diff
@@ -229,15 +221,15 @@ impl SignedTransaction for SignedUserOperationV0_7 {
 struct SignedUserOperationV0_7Serde {
     sender: String,
     nonce: String,
-    factory: String,
+    factory: Option<String>,
     factory_data: String,
```

Comment:
Intuitively, if the factory is optional, then the factory_data should also be optional. 😃